os/include/tinyara/tz_context.h

```diff
@@ -70,9 +70,35 @@ typedef uint32_t TZ_ModuleId_t;
 /* Details TZ Memory ID identifies an allocated memory slot. */
 typedef uint32_t TZ_MemoryId_t;
 
+extern volatile TZ_MemoryId_t tz_memory;
+
+/****************************************************************************
+ * Inline Functions
+ ****************************************************************************/
+ static inline uint32_t get_PSPLIM(void)
```

Comment:
This is not for tz_context. This is the function of stack overflow protection armv8-m provide. Could you move this in proper header?

Reply:
cannot find any specific header for stack. how about arch.h?